packages/manager/modules/pci/src/projects/project/kubernetes/details/service/update/translations/Messages_fr_FR.json

```diff
@@ -2,7 +2,13 @@
   "kube_service_update_common_confirm": "Confirmer",
   "kube_service_update_common_cancel": "Annuler",
   "kube_service_update_title": "Forcer la mise à jour maintenant",
+  "kube_service__minor_version_update_title": "Passer à la version mineure suivante de Kubernetes",
   "kube_service_update_message": "La mise à jour entrainera possiblement une indisponibilité de l'ordre d'une à deux minutes pour chacun de vos noeuds. Nous utilisons la mise à jour roulante (rolling upgrade with node drain) pour rendre l'impact sur vos services exposés minime ou transparent.",
+  "kube_service_minor_version_update_message_1": "Grâce à cette fonctionnalité, vous pouvez monter votre cluster d'une version mineure à la suivante ({{clusterMinorVersion}} vers {{ nextMinorVersion }}). Avant de lancer cette action nous vous invitons à sauvegarder la configuration de votre cluster et valider que les outils avec lesquels vous travaillez sont compatibles avec cette nouvelle version mineure.",
+  "kube_service_minor_version_update_message_2": "Veuillez également vous assurer que vous avez dimensionné votre cluster avec suffisamment de noeuds de travail pour permettre une montée de version roulante (rolling upgrade). Vous ne pourrez pas revenir à la version mineure précédente.",
+  "kube_service_minor_version_update_message_3": "Afin de minimiser le temps d'indisponibilité de chacun de vos pods et de votre service en général, nous allons évacuer ('drain') sucessivement chacun de vos noeuds, passant au suivant dès que le premier est fonctionnel (état 'ready'). Enfin, nous effectuerons la mise à jour des différents composants admin, dont votre API server",
+  "kube_service_minor_version_update_message_4": "L'ensemble de ces opérations prend généralement plusieurs dizaines de minutes (variable selon la typologie de votre cluster).",
+  "kube_service_minor_version_update_message_5": "Passer à la Version : <strong>{{ nextMinorVersion }}</strong>",
```

Comment:
You should split this message in two parts, in order to remove the `<strong>` tag in this translation file, and so handle the boldness of text directly in HTML

Reply:
This is done :)